### PR TITLE
Fix a broken link in list of ingress controllers

### PR DESCRIPTION
**Pull request by freemankevin**

### CODE REVIEW

**Comment by reylejano** [comment]
The original link is valid
The other links in the ingress controller list, list to the product/project page or the github page -- this aligns with the current link for nginx ingress controller. 
This PR updates the link to the docs page of the nginx ingress controller. I prefer the original since it matches with the rest of the list

**Comment by freemankevin** [comment]
https://www.nginx.com/products/nginx-ingress-controller/
image

**Comment by mengjiao-liu** [comment]
> https://www.nginx.com/products/nginx-ingress-controller/ image

Yes, and under my test, several URLs of this page with `https://www.nginx.com/` prefix all return 403.

invalid links: 
https://www.nginx.com/resources/glossary/nginx/
https://www.nginx.com/products/nginx-ingress-controller/

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * The [Kong Ingress Controller for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller#readme)
   is an ingress controller driving [Kong Gateway](https://konghq.com/kong/).
 * [Kusk Gateway](https://kusk.kubeshop.io/) is an OpenAPI-driven ingress controller based on [Envoy](https://www.envoyproxy.io).
-* The [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx-ingress-controller/)
+* The [NGINX Ingress Controller for Kubernetes](https://docs.nginx.com/nginx-ingress-controller/)
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
```
Comment (mengjiao-liu) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:55]: Link https://www.nginx.com/resources/glossary/nginx/ also returns 403.
Please also modify this link.

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:55]: It's also OK to drop the second link

**Comment by sftim** [comment]
/retitle Fix a broken link in list of ingress controllers

**Comment by sftim** [comment]
https://www.nginx.com/products/nginx-ingress-controller/ loads OK for me. Is it possible that corporate filters or state censorship are making that appear like a broken link?

**Comment by sftim** [comment]
Seems like Hong Kong (or maybe just some Hong Kong ISPs) have custom censorship.

/hold

This isn't fixing a bug in Kubernetes.

**Comment by freemankevin** [comment]
> Seems like Hong Kong (or maybe just some Hong Kong ISPs) have custom censorship.
> 
> /hold
> 
> This isn't fixing a bug in Kubernetes.

ok，thanks.

**Comment by sftim** [comment]
/close

Nothing to fix (within Kubernetes)

**Pull request closed without merging**